docs/concepts/Init-Command.md

```diff
@@ -0,0 +1,67 @@
+# INIT COMMAND
+
+`askx init` helps a skill developer to setup a new or an existing Alexa skill project. The command will walk the user through creating an ask-resources.json file to help deploy the skill. It will cover the most common attributes and will suggest sensible defaults using AWS Lambda as your endpoint.
+
+**STRUCTURE OF INIT COMMAND:**
+
+`askx init [-p|--profile <profile>] [--debug]`
+
+**OPTIONS DESCRIPTION:**
+
+**profile**: Optional. Specify a profile name to be used. Defaults to `ASK_DEFAULT_PROFILE`.
```

Comment:
-> `Defaults to use "default" as the profile name, if this option or environmental variable "ASK_DEFAULT_PROFILE" is not set.`